README.md

```diff
@@ -79,27 +82,37 @@ get_taxa(birds)
 ```
 
 ```
-##  [1] "Struthioniformes" "Tinamiformes"     "Craciformes"     
-##  [4] "Galliformes"      "Anseriformes"     "Turniciformes"   
-##  [7] "Piciformes"       "Galbuliformes"    "Bucerotiformes"  
-## [10] "Upupiformes"      "Trogoniformes"    "Coraciiformes"   
-## [13] "Coliiformes"      "Cuculiformes"     "Psittaciformes"  
-## [16] "Apodiformes"      "Trochiliformes"   "Musophagiformes" 
-## [19] "Strigiformes"     "Columbiformes"    "Gruiformes"      
-## [22] "Ciconiiformes"    "Passeriformes"
+## Source: local data frame [23 x 5]
+## 
+##      otu            label about xsi.type  otus
+##    <chr>            <chr> <chr>    <lgl> <chr>
+## 1    ou1 Struthioniformes  #ou1       NA   os1
+## 2    ou2     Tinamiformes  #ou2       NA   os1
+## 3    ou3      Craciformes  #ou3       NA   os1
+## 4    ou4      Galliformes  #ou4       NA   os1
+## 5    ou5     Anseriformes  #ou5       NA   os1
+## 6    ou6    Turniciformes  #ou6       NA   os1
+## 7    ou7       Piciformes  #ou7       NA   os1
+## 8    ou8    Galbuliformes  #ou8       NA   os1
+## 9    ou9   Bucerotiformes  #ou9       NA   os1
+## 10  ou10      Upupiformes #ou10       NA   os1
+## ..   ...              ...   ...      ...   ...
 ```
 
 ```r
 get_metadata(birds) 
 ```
 
 ```
-##                                          dc:creator 
-##                                                  "" 
-##                       dcterms:bibliographicCitation 
-##                                                  NA 
-##                                          cc:license 
-## "http://creativecommons.org/publicdomain/zero/1.0/"
+## Source: local data frame [3 x 8]
+## 
+##   LiteralMeta                      property   datatype content
+##         <chr>                         <chr>      <chr>   <chr>
+## 1          m1                    dc:creator xsd:string    lapp
+## 2          NA                            NA         NA      NA
+## 3          m3 dcterms:bibliographicCitation xsd:string      NA
+## Variables not shown: xsi.type <chr>, ResourceMeta <chr>, rel <chr>, href
+##   <chr>.
```

Comment:
The record with all `NA` (index 2) seems problematic to me - is that kosher and what we're expecting to see? Also, where is the actual value for each property - in `content` (hopefully not, since it's `NA`), or in `href`? I wish we could make this show all the columns.